.github/scripts/pack-interchange-data.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+# Copyright (C) 2022  The SymbiFlow Authors.
```

Comment:
F4PGA Authors

Reply:
Actually we need an Apache license identifier like [here](https://github.com/chipsalliance/fpga-tool-perf/blob/main/.github/scripts/install.sh#L17)

Reply:
fixed